### PR TITLE
Task 4617: Rework Tooltip documentation

### PR DESCRIPTION
**Pull request by sergei820**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -2038,47 +2038,98 @@ For examples of usage see: [Navigation Drawers tests](https://github.com/jdi-tes
...
+</div>
+```
+
+| Method         | Description                                            | Return Type   |
+|----------------|--------------------------------------------------------|---------------|
+| **is()/has()** | Returns Assert class                                   | TooltipAssert |
+| **color()**    | Returns css attribute background-color as String Value | String        |
+
+<a href="https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/TooltipsTests.javas">Java test examples</a>
+
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2132]: Hi, the link does not work :(

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -2038,47 +2038,98 @@ For examples of usage see: [Navigation Drawers tests](https://github.com/jdi-tes
...
+  <div class="v-tooltip__content" style="left: 436px; top: 242px; z-index: 8; display: none;"> <span>Tooltip for "Button"</span></div>
+  <div class="v-tooltip__content" style="left: 313px; top: 309px; z-index: 8; display: none;"> <span>Left tooltip</span></div>
+</div>
+```
+
+| Method         | Description                                            | Return Type   |
+|----------------|--------------------------------------------------------|---------------|
+| **is()/has()** | Returns Assert class                                   | TooltipAssert |
+| **color()**    | Returns css attribute background-color as String Value | String        |
+
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:2130]: Why only two methods? ^^

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 1 above)
Comment (sergei820) [reply source/includes/documentation/_vue_js_elements.md:2132]: Thank you for this comment! It was a misprint, fixed

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (sergei820) [reply source/includes/documentation/_vue_js_elements.md:2130]: Three, is() and has() are in the same line. Should I put them in different lines?

Comment (sergei820) [reply source/includes/documentation/_vue_js_elements.md]: Added all methods that can be used with Tooltip class instances

**Review by MayaElf — APPROVED** [review]

**Pull request merged**